test/integration/sanMvrvRatioTest.js

```diff
@@ -23,14 +25,14 @@ describe('SAN_MVRV_RATIO', () => {
 
```

Comment:
Add a separate test that validates the new function will work correctly without timebound and currency arguments - backward compatible with the old one.